openapi.yaml

```diff
@@ -9259,3 +9341,149 @@ components:
       - account
       - work_type
       type: object
+    DeploymentMetricID:
+      description: |
+        Supported deployment metrics.
+        * `dep-count` - number of deployments in the time period.
+        * `dep-duration` - average deployment procedure time - the difference between `date_finished` and `date_started` of `DeploymentNotification`.
```

Comment:
of successful deployments?

Reply:
Good question. Maybe yes, because failed deployments are often outliers and last much longer.  Shall we then add `dep-duration-all`, `dep-duration-successful`, `dep-duration-failed`?

Reply:
Yes, let's compute it only on successful deployments then because I feel this is going to skew the metric otherwise. We should in theory review how we do it for GH checks as well then.
The declinations of `dep-duration` are interesting even though not a high priority. If that's low overhead to add it now, I'd say yes let's add them.